app/views/clients/clients.html

```diff
@@ -3,10 +3,16 @@
         <li class="active">{{'label.anchor.clients' | translate}}</li>
     </ul>
     <api-validate></api-validate>
-
-    <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="marginbottom0px form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
-    <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
-
+    <div class="col-sm-5">
+        <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
+    </div>
+        <form ng-submit="search()">
+            <div class="col-sm-4">
+            <input  data-ng-model="client.query" type="text" placeholder="Search Client with Name/ExternalId/Identifier/Acc.No" value="{{query}}" class="form-control"/>
+            </div>
+            <button id="query" type="search" class="btn btn-primary" has-permission='READ_CLIENT'>Search Client</button>
+            <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
+        </form>
     <table class="table">
```

Comment:
The look and feel of the new client page is very confusing

1) How would I return to the list of all clients, I would expect clearing the search box and triggering the search afresh to do so
2) Filter does not work after doing a search (As clients displayed from a search are in a different table fragment)